rasterio/__init__.py

```diff
@@ -308,19 +309,24 @@ def pad(array, transform, pad_width, mode=None, **kwargs):
 
 
 def get_data_window(arr, nodata=None):
-    warnings.warn("Deprecated; Use rasterio.windows instead", DeprecationWarning)
+    warnings.warn(
+        "Deprecated; Use rasterio.windows instead", DeprecationWarning)
```

Comment:
@sgillies would help to be more specific here and below: `Deprecated: Use rasterio.windows.get_data_window() instead`